catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -144,13 +145,24 @@ public UpdateRequest process(UpdateRequest input)
             }
         }
 
+        Attribute validationErrors = metacard.getAttribute(Validation.VALIDATION_ERRORS);
+        Attribute validationWarnings = metacard.getAttribute(Validation.VALIDATION_WARNINGS);
+
+        if (validationErrors != null) {
+            errors.addAll(validationErrors.getValues());
```

Comment:
@brendan-hofmann  you may want to double check me here. I wasn't sure if we had a configuration to make warnings be `valid` rather than `invalid`